numpy/testing/_private/utils.py

```diff
@@ -2654,8 +2653,7 @@ def _parse_size(size_str):
                 'kb': 1000, 'mb': 1000**2, 'gb': 1000**3, 'tb': 1000**4,
                 'kib': 1024, 'mib': 1024**2, 'gib': 1024**3, 'tib': 1024**4}
 
-    size_re = re.compile(r'^\s*(\d+|\d+\.\d+)\s*({0})\s*$'.format(
-        '|'.join(suffixes.keys())), re.I)
+    size_re = re.compile(fr'^\s*(\d+|\d+\.\d+)\s*({"|".join(suffixes.keys())})\s*$', re.I)
```

Comment:
Nit: I'd probably lift this expression out to a variable assignment, then use the f-string referring to the variable name. Simpler expressions like `indents[k]` in an earlier file are probably fine, but in a regex, which is already a second problem, this is a pileup of punctuation.

I get it if you really want this PR to be purely mechanical since the tool is doing a pretty good job all around, but I think this one is worth it.

Reply:
I agree with @rkern, I'll add a variable assignment.